README.rst

```diff
@@ -157,3 +157,70 @@ contains metadata about a docker repository.
 
 These files are produced by a publish action in
 `Pulp <http://www.pulpproject.org>`_.
+
+Crane Admin
+-----------
+
+List of images served by Crane can be obtained by opening ``/crane/repositories`` in web browser or
+with ``curl``. The default Apache configuration distributed with Crane restricts access to this URL
+from ``localhost`` only. When accessed from a web browser, a page listing all repositories and
+their images is returned. This URL accepts a "Content-Type" header. When "application/json" is
+specified, the application responds with JSON that has the following format:
```

Comment:
s/responds/replies/
/JSON that has the following format/JSON. Here is an example/